src/com/company/Main.java

```diff
@@ -33,44 +55,34 @@ public static String readFileAsString(String fileName)
               } catch (IOException e) {
                   e.printStackTrace();
               }
-              return data;
+              return data.replaceAll("[^a-zA-Z0-9@]", " ");
```

Comment:
The regex pattern can be made final/constant

Reply:
done with that.